src/core/Select/Select.tsx

```diff
@@ -289,7 +289,7 @@ export const Select = <T,>(props: SelectProps<T>): JSX.Element => {
         style={{
           minWidth,
           maxWidth: `min(${minWidth * 2}px, 90vw)`,
-          maxHeight: `300px`,
+          maxHeight: 297,
```

Comment:
I think height should really be that half of item would be visible 😀 I know there were user raised issues in Android or iOS with share menu or somewhere else that it is not clear that there are more items:
![image](https://user-images.githubusercontent.com/36186912/156199411-0337e2fd-eb03-4d36-b628-170ab7c536df.png)
But maybe it is ok when scroll is visible.

Reply:
i agree
![image](https://user-images.githubusercontent.com/9084735/156200206-6c1250b2-3aa9-40f0-9a22-888b1c134673.png)
![image](https://user-images.githubusercontent.com/9084735/156200234-37320481-75d5-44c6-b49c-1cb0263c4e02.png)

Reply:
Then you are both free to take over this PR and fix this 😄

Reply:
my calculations were messed up when taking container height and calculating amount of items to fit in..
I guess I can try to change that now and see, maybe this one also magically starts working just fine 😟

Reply:
Ok, changed max height but didnt hack with the scroll down..
Check it up